atd-vze/src/helpers/map.js

```diff
@@ -111,13 +113,43 @@ export const LOCATION_MAP_CONFIG = {
   },
 };
 
-export const LabeledAerialSourceAndLayer = () => {
-  return (
+const mockGeoJSON = {
+  type: "Feature",
+  properties: {},
+  geometry: {
+    type: "Polygon",
+    coordinates: [[[-99, 29], [-96, 29], [-96, 32], [-99, 32], [-99, 29]]],
+  },
+};
+
+const mockPolygonDataLayer = {
+  id: "test-location-polygon",
+  type: "fill",
+  paint: {
+    "fill-color": colors.primary,
+    "fill-opacity": 0.3, // values 0 to 1
+  },
+};
+
+/** Source and layer to display NearMap aerials with street labels on top.
+ * Localhost and deploy preview URLs are not on the NearMap tile API key allow list.
+ * To test locally, this returns a mock layer to test the layer ordering.
+ * Adjust the opacity in the mockPolygonDataLayer paint object for further testing.
+ * @param {string} beforeId - layer id to place these layers before
+ */
+export const LabeledAerialSourceAndLayer = ({ beforeId }) => {
+  return isDev ? (
```

Comment:
we were conditionally rendering this component in each use, but I think it makes more sense for it to consider environment itself now. idk

Reply:
yes—i agree!